qualang_tools/plot/Fitting/fitting_functions.py

```diff
@@ -0,0 +1,348 @@
+from scipy import optimize
+import numpy as np
+
+
+def fit_ramsey(x, y):
+    """
+    Create a fit to Ramsey experiment of the form
+
+    .. math::
+    f(x) = uncertainty_population * (1 - np.exp(-x * (1/tau))) + amp / 2 * (
+        np.exp(-x * (1/tau))
+        * (initial_offset * 2 + np.cos(2 * np.pi * f * x + phase))
+        )
+
+    for unknown parameters `f`, `phase`, `tau`, `amp`, `uncertainty_population` and `initial_offset`.
+
+    :param x:  The data on the x axis
```

Comment:
This should be the dephasing time right?

Reply:
Yes, in seconds or ns?

Reply:
Then I would call it dephasing time instead of The data on the x axis and I think that the best is to have it in ns because this is the unit it usually is in the QUA program. Be careful that in this case the frequency will be in GHz so we may want to convert it in MHz